recipes/scikit-posthocs/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "scikit-posthocs" %}
+{% set version = "0.6.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/scikit-posthocs-{{ version }}.tar.gz
```

Comment:
Installing from the source distribution tarball is broken due to a missing file. I've attempted to fix this as well as distribution of the license file in https://github.com/maximtrp/scikit-posthocs/pull/36

You need to either push out a new release of the package that fixes this or install from a Github release. I think the former is preferable, but I'll leave it up to you since I don't know your release cycle constraints.

Reply:
I have pushed out a new release to PyPi as you suggested. Thank you!

Reply:
Thanks for merging and making the update. Currently failing since the sha256 needs to be updated for the new release.

Reply:
Sorry! Just updated it